e2e/tests/wasm/grandpa_test.go

```diff
@@ -96,14 +104,6 @@ func (s *GrandpaTestSuite) TestMsgTransfer_Succeeds_GrandpaContract() {
 		options.ChainBSpec.Type = "cosmos"
 		options.ChainBSpec.Name = "simd"
 		options.ChainBSpec.ChainID = simd
-		options.ChainBSpec.Images = []ibc.DockerImage{
```

Comment:
these values are now configured by reading from the test config (which is updated to point at the image built on PRs)